.rubocop.yml

```diff
@@ -0,0 +1,54 @@
+AllCops:
+  Include:
+    - '**/*.rb'
+    - 'archive_posts.rake'
+  Exclude:
+    - bin/**/*
+    - db/**/*
+    - node_modules/**/*
+  TargetRubyVersion: 2.6.3
+
+# Rubocop Style Choices
+
+Layout/AccessModifierIndentation:
+  EnforcedStyle: outdent
+
+Layout/HashAlignment:
+  EnforcedLastArgumentHashStyle: ignore_implicit
+
+Layout/LineLength:
+  Max: 240
+
+Metrics/BlockLength:
+  ExcludedMethods: ["context", "describe", "shared_examples"]
+
+Style/ClassAndModuleChildren:
+  EnforcedStyle: compact
+  Exclude:
+    - config/application.rb
+    - app/channels/application_cable/channel.rb
+    - app/channels/application_cable/connection.rb
+
+Style/Documentation:
+  Enabled: false
+
+Style/EmptyMethod:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/HashEachMethods:
+  Enabled: false
+
+Style/HashTransformKeys:
+  Enabled: false
+
+Style/HashTransformValues:
+  Enabled: false
+
+Style/MultilineTernaryOperator:
+  Enabled: false
```

Comment:
Why did you choose this cop?

Reply:
I took lead from you! It made some nice improvements to readability in my spec files :)

Reply:
Is it necessary in your project? What happens when you comment out this cop?

Reply:
There are no offences right now but if I checkout the initial commit `git checkout f4d064b` I can test.

After I run rubocop, comment this cop out and run rubocop again, I have the same number of offences. So this is not a cop I was in violation of.